### DISCUSSION TITLE
Overlapping a texture fiber ODF on a pole figure

### DISCUSSION THREAD

**Discussion by AdamLeaghj** [mtex-toolbox/mtex#1747] [Ask Anything] [answered]
Hi, 

I have a new plot of pole figure and wanted to overlap a pre-defined texture fiber on it. I've used the code as follows but it seems doesn't work as expected. In addition, the idea is not to show the intensity of pre-defined ODF on the pole figure plot but just wanted to indicate the tolerance angle for ODF calculation (in such case is 5 degrees). 
Any idea on this would be much appreciated. Thank you.

```
close all;
figure();
ori=ebsd('Martensite BCC').orientations;
x=[Miller(1,1,2,ori.CS)]; 
plotPDF(ori,x,'antipodal', 'contourf', 1:0.6:5, 'colorRange',[0,5],'minmax'); 
hold on;
ori_bcc.SS = specimenSymmetry('triclinic');
f = fibre(Miller(1,1,1,ebsd('Martensite BCC').CS),vector3d(0,0,1),ori_bcc.SS,'full');
odf = fibreODF(f,'halfwidth',5*degree)
plotPDF(odf,Miller({1,1,2},ebsd('Martensite BCC').CS))
plotPDF(f.symmetrise,Miller({1,1,2},ebsd('Martensite BCC').CS),'linewidth',3,'lineColor','orange','antipodal')
hold off;
```

![image](https://user-images.githubusercontent.com/118141541/221323972-66b394e2-beaa-4e14-9976-c469796ff81b.png)

**Comment by kilir** [comment, marked as answer]
Hi,
I'm not really sure if I understand what you mena with "showing the tolerance angle of 5 degree". Maybe you can elaborate. Second, is it intentional that your overlay plot should just cover 1/4 of the hemisphere?
Cheers,
Rdiger

**Reply by AdamLeaghj** [reply]
Hi Rudiger,

I think I figured it out after a few trials. Please help to remove this thread if possible. Thank you for your follow-up answer!
